repos/base-hw/src/core/include/spec/x86_64/muen/timer.h

```diff
@@ -63,6 +63,10 @@ class Genode::Timer
 		Timer() :
 			_tics_per_ms(sinfo()->get_tsc_khz())
 		{
+
+			/* first sinfo instance, output status */
+			sinfo()->log_status();
+
```

Comment:
I want the memory/channel info logged only once and decided in favor of an explicit call to `log_status()` at the first instance.